tests/cli/data/test_prop2po_files/stderr.txt

```diff
@@ -0,0 +1 @@
+root: WARNING: trying to parse ./data/test_prop2po_files/one.properties with encoding: iso-8859-1 but detected encoding is utf-8 (confidence: 1.0)
```

Comment:
Thinking more about this, this warning is really wrong in this case. I've tried to fix that in 548445de92ed5928898cdbd45741c34503667468.